cf/regrid/utils.py

```diff
@@ -258,12 +258,15 @@ def regrid_get_latlon(f, name, axes=None):
         coords_2D = True
     else:
         coords_2D = False
-        # Check for size 1 latitude or longitude dimensions
-        if x_size == 1 or y_size == 1:
-            raise ValueError(
-                "Neither the longitude nor latitude dimension coordinates "
-                f"of the {name} field can be of size 1."
-            )
+
+        # Check for size 1 latitude or longitude dimensions if source grid
+        # (a size 1 dimension is only problematic for the source grid in ESMF)
+        if name == "source":
+            if x_size == 1 or y_size == 1:
+                raise ValueError(
+                    "Neither the longitude nor latitude dimension coordinates "
+                    f"of the {name} field can be of size 1."
+                )
```

Comment:
I'm wondering why this highlighted block of code is inside the "else" clause, and is not indented one tab to the left ... This has always been the case, but looking at it afresh it'd be nice to know why the check is only done when `coords_2D = False`, and if that's correct.

Reply:
This is good now, I think